### DISCUSSION TITLE
Add to queue feature request.

### DISCUSSION THREAD

**Discussion by Huecuva** [FreeTubeApp/FreeTube#6381] [General]
Hi. There's another feature I would like to request. I have searched for this and could find nothing related. I also could not find a similar feature in FreeTube.

It would be great to be able to add videos to a queue without creating a playlist, as one is able to in YouTube. Like a temporary playlist. Again, correct me if this already exists, but I could not find anything of the sort. Also, perhaps the ability to then save the created queue as a playlist. That would be handy as well. I'm pretty sure this is also possible in YouTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of #547

**Comment by Huecuva** [comment]
Man, I searched for "Add to queue", "play next" and even just "queue" and couldn't find anything. You'd think just searching for "queue" would have found that.
